src/pyciemss/PetriNetODE/interfaces.py

```diff
@@ -231,6 +231,321 @@ def load_and_calibrate_and_sample_petri_model(
     return processed_samples
 
 
+def load_and_optimize_and_sample_petri_model(
+    petri_model_or_path: Union[str, mira.metamodel.TemplateModel, mira.modeling.Model],
+    num_samples: int,
+    timepoints: Iterable[float],
+    interventions: Iterable[Tuple[float, str]],
+    qoi: Iterable[Tuple[str,str,float]],
+    risk_bound: float,
+    objfun: callable = lambda x: np.abs(x),
+    initial_guess: Iterable[float] = 0.5,
+    bounds: Iterable[float] = [[0.0], [1.0]],
+    *,
+    start_state: Optional[dict[str, float]] = None,
+    pseudocount: float = 1.0,
+    start_time: float = -1e-10,
+    method="dopri5",
+    verbose: bool = False,
+    n_samples_ouu: int = int(1e2),
+    maxiter: int = 2,
+    maxfeval: int = 25,
+) -> pd.DataFrame:
```

Comment:
This return type is not quite right anymore. Could you change this to `Tuple[pd.DataFrame, TODO]` where TODO is the type of the ouu_policy?

Reply:
Done